server/chat-commands/room-settings.ts

```diff
@@ -922,6 +923,12 @@ export const commands: ChatCommands = {
 			if (this.filter(target) !== target) {
 				return this.errorReply("Invalid title.");
 			}
+			// `,` is a delimiter used by a lot of /commands
+			// `|` and `[` are delimiters used by the protocol
+			// `-` has special meaning in roomids
+			if (target.includes(',') || target.includes('|') || target.includes('[') || target.includes('-')) {
+				return this.errorReply("Room titles can't contain any of: ,|[-");
+			}
```

Comment:
```suggestion
```

This restriction isn't enforced in `/creategroupchat` anyway. I don't think it's particularly a big deal.

Reply:
Apologies if I'm misreading this, but it does enforce that? I can't currently create groupchats with those special characters in their titles on main.

Reply:
Most of the checks here are redundant with `validateTitle`. The main restriction that's been lifted is that `[` is no longer banned.

Reply:
Wait, apparently that's still there. Ugh, I guess this will need a separate fix.